src/app/business/delfin/delfin.service.ts

```diff
@@ -152,5 +152,10 @@ export class DelfinService {
     deleteAlerts(id, seqNum): Observable<any>{
         return this.http.delete(this.delfinStoragesUrl + '/' + id + '/alerts/' + seqNum);
     }
+
+    //Configure performance metrics collection
+    metricsConfig(id, params): Observable<any>{
```

Comment:
Few comments on the congiguration page
- change scrape interval to polling interval , also mention the value is in seconds 
- Change 'Enable History' to Enable Historic metric collection, ( also if possible provide a tool tip like ' historic collection is the collection all availabe timestamps from previous polling to current polling, also supported only if the backend supports historic collection'

Reply:
Do you have any change for above comment, could not see it in screen shots..

Reply:
@NajmudheenCT  These are updated. 
Screenshot below.
![image](https://user-images.githubusercontent.com/19162717/102451976-8e653400-405f-11eb-874c-aa189062dc42.png)

Reply:
@NajmudheenCT I believe the changes have gone in a subsequent PR #481 
I will update this PR to add the changes.

Reply:
Updated the PR.